Resources/Prototypes/Catalog/uplink_catalog.yml

```diff
@@ -240,7 +240,7 @@
       - NukeOpsUplink
 
 - type: listing
-  id: UplinkSyndicateBombNukie
```

Comment:
We don't have prototype migrations, just mapping ones so this should be left as is.

Reply:
Alright, thank you. I've reverted that change just now.